doc/explanation/api/reactivity.md

```diff
@@ -199,12 +203,13 @@ template = pn.rx('Hello {text}!')
 
 pn.pane.Markdown(object=template.format(text=text))
 ```
+
 :::
 
 :::{tab-item} (Async) Generators
 (Asynchronous) generators functions can be used as a reference, to drive streaming outputs.
 
-```{pyodide}
+```python
```

Comment:
This was blocking, as far as I could see.

Reply:
There are other places where this happens. Looks like a regression.

Reply:
Thanks, will investigate.

Reply:
Likely needs a change in `nbsite` to kill running background tasks.